Flow.Launcher/SettingWindow.xaml.cs

```diff
@@ -386,33 +394,41 @@ private void OnDeleteCustomShortCutClick(object sender, RoutedEventArgs e)
                 MessageBox.Show(deleteWarning, InternationalizationManager.Instance.GetTranslation("delete"),
                     MessageBoxButton.YesNo) == MessageBoxResult.Yes)
             {
-                settings.ShortCuts.Remove(item.Value);
+                viewModel.RemoveShortcut(item);
             }
         }
+
         private void OnEditCustomShortCutClick(object sender, RoutedEventArgs e)
         {
             var item = viewModel.SelectedCustomShortcut;
-            if (item != null)
+            if (item == null)
             {
-                var shortcutSettingWindow = new CustomShortcutSetting(item.Value);
-                if (shortcutSettingWindow.ShowDialog() == true)
-                {
-                    settings.ShortCuts[viewModel.SelectCustomShortcutIndex.Value] = shortcutSettingWindow.ShortCut;
-                }
+                MessageBox.Show(InternationalizationManager.Instance.GetTranslation("pleaseSelectAnItem"));
+                return;
             }
-            else
+            else if (!item.CanBeEdited)
```

Comment:
Let's remove this check. The buildin shortcut is more like a variable that is dynamic. We may want to display them with their key and description (not just the value).

Although I am thinking there may be snippet plugins somewhat like the vscode that only provides a bunch of snippets, but I also think the description can be the value. Therefore, let's make them separate and be clear about everything.